include/libdnf/base/goal.hpp

```diff
@@ -36,8 +34,8 @@ class Goal {
 public:
     struct UsedDifferentSack : public LogicError {
         using LogicError::LogicError;
```

Comment:
The `UsedDifferentSack()` constructor is defined. The line `using LogicError::LogicError;` is no longer needed.